src/pybind/mgr/zabbix/module.py

```diff
@@ -0,0 +1,209 @@
+"""
+Zabbix module for ceph-mgr
+
+Collect statistics from Ceph cluster and every X seconds send data to a Zabbix
+server using the zabbix_sender executable.
+"""
+import json
+import subprocess
+from Queue import Queue, Empty
+from mgr_module import MgrModule
+
+
+class ZabbixSender(object):
+    def __init__(self, sender, host, port, log):
+        self.sender = sender
+        self.host = host
+        self.port = port
+        self.log = log
+
+    def send(self, hostname, data):
+        if len(data) == 0:
+            return
+
+        try:
+            cmd = '{0} -z {1} -p {2} -s {3} -vv -i -'.format(self.sender,
+                                                             self.host,
+                                                             self.port,
+                                                             hostname)
+
+            proc = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE,
```

Comment:
Avoid using shell=True if possible -- passing in a `[]` of arguments avoids vulnerability to malicious argument substitution in the string.